hack/auto_generate_charts.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+
+# receive version version
+echo "----------------------------------"
+echo "Now auto version"
```

Comment:
How about this 
echo "-------------------------------"
echo "Please input Kruise version (available versions are listed in https://github.com/openkruise/kruise/releases)"
echo "If no input, the default version of v0.1.0 will be used"
echo "-------------------------------"